README.md

```diff
@@ -202,6 +202,16 @@ Documentation for using each command is available by typing:
 > python manage.py help <command>
 ```
 
+### Pickle Tribe gene sets
+
+This step downloads and pickles all the public Tribe gene sets for every
+organism that has been loaded into the database. These pickled files are
+necessary for doing gene set enrichment analyses.
```

Comment:
I know you're adding it to the `tribe_client` documentation, but do you want to make a note here that this command depends upon some settings? Maybe a link directly to the `tribe_client` docs would be the right thing. I'll go review those changes now.